vm.go

```diff
@@ -64,6 +66,15 @@ func El(selector string) ComponentOption {
 	}
 }
 
+// Store enables the Vuex store option, as described here: https://vuex.vuejs.org/guide/state.html
+func Store() ComponentOption {
```

Comment:
Indeed you can initialize a Store everywhere and name it as you like, but my test shows that you wouldn't be able to inject it into Vue.

This seems to be, becaues the option is provided to Vue without a value, see snippet from https://vuex.vuejs.org/guide/state.html

```
const app = new Vue({
  el: '#app',
  // provide the store using the "store" option.
  // this will inject the store instance to all child components.
  store,  //<-- doesn't allow to provide a value, in contrast to the other options
  components: { Counter },
  template: `
    <div class="app">
      <counter></counter>
    </div>
  `
})
```

Anyway, rethinking everything, there seems to be no need to merge thi PR in order to get it working, as you exported the `ComponentOption` struct. Thus it should be perfectly possible to provide a custom component option from external code.

Reply:
I see ... guess I'm not really awake.

So I released an example, showing my Vuex bindings combined with your Vue bindings.
It uses an external Method to set the `Store` option in the fashion you proposed, here:

https://github.com/mame82/mvuex/blob/master/example/main.go#L9

Reply:
You could test the example by running

```
go get -u github.com/mame82/mvuex/example
gopherjs serve
```

... and browsing to: `http://localhost:8080/github.com/mame82/mvuex/example/`

It shows my partial Vuex bindings working pretty well together with your Vue bindings.
Hope somebody (maybe you) grabs the Vuex bindings to add the missing functionality. But as stated, it would rely on promises if done the right way. Implementing all the mapping functionality (mapState, mapGetters, etc) would require close integration into hvue (which is much more capable than gopher-js).